M/ModularForms/build_tarballs.jl

```diff
@@ -29,8 +33,7 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency(PackageSpec(name="Arb_jll"), compat = "~200.2200.0"),
-    Dependency(PackageSpec(name="FLINT_jll"), compat = "^200.800.401")
+    Dependency(PackageSpec(name="FLINT_jll"), compat = "~301.300")
```

Comment:
How about going to 3.4 directly?
```suggestion
    Dependency(PackageSpec(name="FLINT_jll"), compat = "~301.400")
```